src/language/from_relay/run_relay.py

```diff
@@ -12,13 +12,12 @@
     action='store_true',
     help='Temporary band-aid b/c nn.dense Relay parser is broken')
 parser.add_argument('--relay_filepath',
-                    nargs=1,
                     type=argparse.FileType('rb'),
                     default=sys.stdin)
-parser.add_argument('npy_out_filepath', type=argparse.FileType('wb'))
-parser.add_argument('npy_arg_filepath',
-                    nargs='*',
-                    type=argparse.FileType('rb'))
+parser.add_argument('--npy_out_filepath', 
+                    nargs='*')
+parser.add_argument('--npy_arg_filepath',
+                    nargs='*')
```

Comment:
Is it not possible to use `type=...` with flag-type arguments in `argparse`? Why did you remove the `type=argparse.FileType(...)`?

Reply:
ah, it was because when I was testing yolo, the test failed on the argument parser.

It turns out that using the `type=...` argument opens all the files at once but having to open so many files at the same time without closing any caused the test to fail because theres a limit to the number of open files at once :p

I'll add a comment to this

Reply:
Haha, what a bug. Good catch. Thanks!